### PR TITLE
Added the 'isInputWithoutSelectionProperties' util method

### PR DESCRIPTION
**Pull request by miherlosev**
@helen-dikareva @LavrovArtem

### CODE REVIEW

File: src/client/sandbox/event/selection.ts [hunk 1]
```diff
@@ -1,17 +1,22 @@
...
 
-const browserResetInputSelection = browserUtils.isFirefox && browserUtils.version > 50;
+const browserResetInputSelection = browserUtils.isFirefox;
 
 export default class Selection {
-    focusBlurSandbox: any;
-    timersSandbox: any;
-    listeners: any;
-    eventSimulator: any;
+    focusBlurSandbox: FocusBlurSandbox;
```
Comment (helen-dikareva) [inline src/client/sandbox/event/selection.ts]: We also have `_isNumberOrEmailInput` in Selection that now used only in `_needChangeInputType`. I suggest moving `_isNumberOrEmailInput` in dom utils and use it in `isInputWithoutSelectionProperties` and in `_needForInternalSelection`.

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

**Pull request merged**